Source/WebGPU/WebGPU/Queue.h

```diff
@@ -88,15 +88,18 @@ class Queue : public WGPUQueueImpl, public ThreadSafeRefCounted<Queue> {
     static bool writeWillCompletelyClear(WGPUTextureDimension, uint32_t widthForMetal, uint32_t logicalSizeWidth, uint32_t heightForMetal, uint32_t logicalSizeHeight, uint32_t depthForMetal, uint32_t logicalSizeDepthOrArrayLayers);
     void endEncoding(id<MTLCommandEncoder>, id<MTLCommandBuffer>) const;
 
+    void ensureBlitCommandEncoder();
```

Comment:
we could change the return type to `id<MTLBlitCommandEncoder>` similar to what CommandEncoder does so you don't need separate `ensure...` and accessor member functions